iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/PhaseTapChangerAdderImpl.java

```diff
@@ -149,6 +151,12 @@ public PhaseTapChangerAdder setRegulating(boolean regulating) {
         return this;
     }
 
+    @Override
+    public PhaseTapChangerAdder setTargetDeadband(double targetDeadband) {
+        this.targetDeadband = targetDeadband;
```

Comment:
Maybe you should add an integrity test also here (not only in setRegulationTerminal).

Reply:
done